geometry/tagged/computeToolpath.js

```diff
@@ -0,0 +1,201 @@
+import KdBush from 'kdbush';
+import { fuse } from './fuse.js';
+import { getNonVoidSegments } from './getNonVoidSegments.js';
+import { getQuery } from '../graph/getQuery.js';
+import { identityMatrix } from '@jsxcad/math-mat4';
+import { inset } from './inset.js';
+import { measureBoundingBox } from './measureBoundingBox.js';
+import { outline } from './outline.js';
+import { section } from './section.js';
+import { taggedToolpath } from './taggedToolpath.js';
+import { toConcreteGeometry } from './toConcreteGeometry.js';
+
+const X = 0;
+const Y = 1;
+const Z = 2;
+
+export const computeToolpath = (geometry, toolDiameter = 1, jumpHeight = 1) => {
```

Comment:
This is impressively compact for everything that it's doing

Reply:
Yeah, but it's also doing things very inefficiently. :)

I suspect it'll get a bit larger.